build.js

```diff
@@ -3,15 +3,15 @@ var exec = require('child_process').execSync;
 var fs = require('fs');
 var path = require('path');
 
-var excludedDirectories = ['css', 'images', 'dist','node_modules','webdriver-java'];
-
 // set the following variable to resume building with a framework and skip all
 // other frameworks that would be buils before
 var restartWithFramework = '';
 var build = !restartWithFramework ? true : false;
 
-_.each(fs.readdirSync('.'), function(name) {
-	if(!name.startsWith(".") && fs.statSync(name).isDirectory() && excludedDirectories.indexOf(name)==-1 && fs.existsSync(path.join(name, "package.json"))) {
```

Comment:
[`readdirSync`](https://nodejs.org/docs/latest-v8.x/api/fs.html#fs_fs_readdirsync_path_options) excludes the `.` entry